src/cljs/rems/administration/organization.cljs

```diff
@@ -71,13 +71,12 @@
                                                                       (map enrich-user)
                                                                       (map :display)
                                                                       (interpose [:br]))]
-              [info-field
-               (text :t.administration/review-emails)
-               (->> (:organization/review-emails organization)
-                    (map display-localized-review-email)
-                    (interpose [:br]))
-               {:box? false
-                :solid? true}]
+              [:div
+               [:label (text :t.administration/review-emails)]
+               [:div.solid-group
+                (->> (:organization/review-emails organization)
+                     (map display-localized-review-email)
+                     (interpose [:br]))]]
```

Comment:
The original intent here was to move into a direction where our detail views i.e. info-fields could be nested. Also sequences should be handled by rows or maps as tables. I think this moves a bit more to the hand-written direction but clearly there is no complete solution for the nesting yet.

Reply:
yeah... I thought about changing slurp-fields so that it only looks at the immediate children, but that was difficult and would've been fragile in another way

Reply:
Something to keep in mind for later as the model gets more complex.

Reply:
I guess one solution would be to look for all the text boxes and then get their labels, instead of looking for form-groups

Reply:
...in slurp-fields that is